### ISSUE TITLE
gulp file broken (gulp-util not installed)

### ISSUE THREAD

**Issue by treyhunner** [startersacademy/fullstack-project-01#38]
Attempting to run the tests gives me an error in the gulp file:
```
$ gulp integrate
module.js:340
    throw err;
          ^
Error: Cannot find module 'gulp-util'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/trey/repos/project/gulpfile.js:6:13)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

**Comment by jeffreytu** [comment]
It's no longer necessary. I was the only one using it.

**Comment by goldlilys** [comment]
@jeffreytu  Does that mean the spawn and other integrate:win is not necessary too ?

**Comment by jeffreytu** [comment]
I still need child_process spawn. The integrate:win will be gone in my pull request this week.

**Comment by jeffreytu** [comment]
Issue fixed in #49.
